### DISCUSSION TITLE
Admin dashboard doesn't always shot title of media being played

### DISCUSSION THREAD

**Discussion by damarges** [jellyfin/jellyfin#13009] [General] [answered]
While users are playing media in admin dashboard I only see title for media, that has no "name image" as I would call it (don't know the exact description). See a screenshot here:
![image](https://github.com/user-attachments/assets/f4293549-4da5-49de-a913-607c8e3f4c06)

Is that me? User error? Or is it by jellyfin server? I'm using version 10.10.1

You see on the left if I am lucky I can decipher the title of the movie being watched ... on the right I can see the title right away.

**Comment by crobibero** [comment]
The web client displays an image instead of the name, you aren't missing anything.

**Reply by damarges** [reply]
Okay, so it's not me using it wrong. Sometimes the images are so unclear or with details that you can't read the name. Can I switch this behaviour so it always display the title being played in clear text instead of this image?

**Reply by crobibero** [reply]
No, that is not configurable. You would need to edit the web client.

**Reply by crobibero** [reply, marked as answer]
No, that is not configurable. You would need to edit the web client.

**Reply by damarges** [reply]
thank you for the reply and clarification :)
